src/module/main.cpp

```diff
@@ -700,8 +700,8 @@ class Orbbec : public vsdk::Camera, public vsdk::Reconfigurable {
     }
 
     std::vector<vsdk::GeometryConfig> get_geometries(const vsdk::ProtoStruct& extra) {
-        VIAM_SDK_LOG(error) << "get_geometries not implemented";
-        return std::vector<vsdk::GeometryConfig>{};
+        // see https://github.com/viam-modules/orbbec/pull/16 for explantation of geometry config values
```

Comment:
please also add a comment that if we add support for other models we need to have un hardcode this.